src/test/groovy/graphql/execution/ExecutionStrategyExceptionHandlingEquivalenceTest.groovy

```diff
@@ -0,0 +1,63 @@
+package graphql.execution
+
+import graphql.ExecutionInput
+import graphql.GraphQL
+import graphql.StarWarsSchema
+import graphql.execution.instrumentation.InstrumentationContext
+import graphql.execution.instrumentation.SimpleInstrumentation
+import graphql.execution.instrumentation.parameters.InstrumentationFieldFetchParameters
+import graphql.validation.ValidationError
+import graphql.validation.ValidationErrorType
+import spock.lang.Specification
+import spock.lang.Unroll
+
+/**
+ * This allows the testing of different execution strategies that handle exceptions during data fetching in the same way
+ */
+class ExecutionStrategyExceptionHandlingEquivalenceTest extends Specification {
+
+  class TestInstrumentation extends SimpleInstrumentation {
+
+    @Override
+    InstrumentationContext<Object> beginFieldFetch(InstrumentationFieldFetchParameters parameters) {
+      throw new AbortExecutionException(Arrays.asList(new ValidationError(ValidationErrorType.UnknownType)))
```

Comment:
We just picked a random error that implemented `GraphQLError` here, but I'm happy to add a generic error to test on instead.